Xext/dpms.c

```diff
@@ -331,28 +321,26 @@ ProcDPMSGetVersion(ClientPtr client)
 static int
 ProcDPMSCapable(ClientPtr client)
 {
-    /* REQUEST(xDPMSCapableReq); */
+    X_REQUEST_HEAD_STRUCT(xDPMSCapableReq);
+
     xDPMSCapableReply reply = {
         .capable = TRUE
     };
 
-    REQUEST_SIZE_MATCH(xDPMSCapableReq);
-
     return X_SEND_REPLY_SIMPLE(client, reply);
 }
 
 static int
 ProcDPMSGetTimeouts(ClientPtr client)
 {
-    /* REQUEST(xDPMSGetTimeoutsReq); */
+    X_REQUEST_HEAD_STRUCT(xDPMSGetTimeoutsReq);
+
     xDPMSGetTimeoutsReply reply = {
         .standby = DPMSStandbyTime / MILLI_PER_SECOND,
         .suspend = DPMSSuspendTime / MILLI_PER_SECOND,
         .off = DPMSOffTime / MILLI_PER_SECOND
     };
 
-    REQUEST_SIZE_MATCH(xDPMSGetTimeoutsReq);
```

Comment:
Same as in the previous function - do we need to switch to the new macros when originally `REQUEST` wasn’t needed?